src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -2931,7 +2931,8 @@ Well{0} entered with 'FIELD' parent group:
             { "WELOPEN" , &Schedule::handleWELOPEN   },
             { "WELPI"   , &Schedule::handleWELPI     },
             { "WELSEGS" , &Schedule::handleWELSEGS   },
-            { "WELSPECS", &Schedule::handleWELSPECS  },
+            { "WELSPECS", &Schedule::handleWELSPEC   },
+            { "WELSPECL", &Schedule::handleWELSPEC   },
```

Comment:
Tiny nit: Could you keep the list in sorted order ("SPECL" before "SPECS"), please?

Reply:
Done.